ui/components/ui/custom-nonce/index.scss

```diff
@@ -0,0 +1,21 @@
+.custom-nonce {
+  &__content {
+    height: 49px;
+    border: 1px solid var(--color-border-muted);
+    box-sizing: border-box;
+  }
+
+  &__header {
+    flex: 1;
+  }
+
+  &__value {
+    flex: 0;
+  }
+
+  &__edit {
+    @include H7;
+
+    width: auto;
```

Comment:
It should be possible to get rid of `@include H7;` with new design components.

Reply:
@jpuri I used `<ButtonLink/>` component for displaying `Edit` button for custom nonce. You can check that [here](https://github.com/MetaMask/metamask-extension/pull/17945/commits/c493916a34f2ff6cb424b823f64cd5f0bad9385b).